libqf/libqfqmlwidgets/src/tableview.cpp

```diff
@@ -2067,12 +2067,12 @@ void TableView::removeSelectedRowsInline()
 
 bool TableView::edit(const QModelIndex& index, EditTrigger trigger, QEvent* event)
 {
-	qfLogFuncFrame() << "trigger:" << trigger << "event type:" << ((event)? event->type(): 0) << "editTriggers:" << editTriggers();
```

Comment:
Good point.  Although Qt documentation doesn't say anything about event pointer value, grep through Qt sources shows that CurrentChanged trigger comes with NULL event pointer.  That means these checks must stay in place and event->accept() can be called without any checking as it's guarded by the trigger type test.  I'll drop this patch from the series.